package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Helping people see and understand queries.",
   "main": "d3/query-graphs.js",
   "scripts": {
```

Comment:
Need to add d3/tql.js to list of "lint" source files. (Comment here because GitHub doesn't allow commenting the correct line as it's not in the PR.) Run lint `npm run lint`. May want to disable some of the errors/warnings for this file, but not all.

Also, please run `npm run bundle-tlv`, which will fail because the version of uglify we require is not ES6 and tql.js expects ES6 features. I tried using latest version of uglify-js@3, but then there are other complaints. It might be simplest to remove use of default function parameters, *groan*. Or, work through the other ES6 issues using uglify-js@3. Or, use a transpiler, e.g., babel.

Reply:
OK, I fixed the things that were real errors (`eqeqeq`) and suppressed all the complaints about my (more readable) use of spaces. If the file wasn't about 1000 lines with an error on every line I might have been convinced, but it was just too much work.

 I also tried to lint my unit tests, but `eslint` does not understand how `mocha` extends the namespace so I gave up. IMHO, unit tests are more important than format fascism!